modules/apps/29-fee/keeper/grpc_query_test.go

```diff
@@ -210,19 +217,46 @@ func (suite *KeeperTestSuite) TestQueryIncentivizedPacketsForChannel() {
 		{
 			"no packets for specified channel",
 			func() {
+				const emptyChannelID = "channel-10"
+				suite.chainA.App.GetIBCKeeper().ChannelKeeper.SetChannel(
+					suite.chainA.GetContext(),
+					ibctesting.MockFeePort,
+					emptyChannelID,
+					channeltypes.Channel{},
+				)
```

Comment:
Absolutely! To be honest, I totally forgot I left it hanging here https://github.com/cosmos/ibc-go/pull/6069#discussion_r1544634150 and didn't come back to it :) Since it got no objection, I'll go with that!